README.md

```diff
@@ -45,31 +45,55 @@ end
 Then in your controllers where you'd like to record an event:
 
 ```ruby
-Track.track('Did something important',
+Klaviyo::Track.track('Did something important',
```

Comment:
is there a reason we call this Track.track or could we combine identify and track so we could do Public.track or Public.identify?